orc8r/cloud/go/obsidian/obsidian/main.go

```diff
@@ -59,6 +59,12 @@ func main() {
 		log.Fatalf("Error creating service: %s", err)
 	}
 
+	enableRunTimeSpec := srv.Config.MustGetBool(obsidian.EnableRunTimeSpecs)
```

Comment:
I don't think there's a great need to make this config value override-able via CLI flag, so we can remove these additions

Reply:
If this is the case, I'm thinking that we would get rid of the config in obsidian.yml and just have a boolean flag in config.go?